watson-developer-cloud/discovery-components
# feat: script that starts client AND server for example app

#### What do these changes do/fix?
In our manual setup instructions, we never tell the user that they now have to start not only the client but the server as well. This changes the name of the start client script to `start:client`, and adds a new `start` script that runs both the server and the client.

It also fixes I typo I spotted in the Windows Only section of our readme.


#### How do you test/verify these changes?
1. Run `yarn workspace discovery-search-app start:client` and make sure that it starts only the client at localhost:3000.
2. Run `yarn workspace discovery-search-app start` and make sure that it starts both the server (localhost:4000) and the client (localhost:3000).
#### Have you documented your changes (if necessary)?
N/A
#### Are there any breaking changes included in this pull request?
No
